.gitignore

```diff
@@ -158,3 +158,44 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+.yarn/install-state.gz
+
+# testing
+/coverage
+
+# next.js
+/.next/
+frontend/.next/
+/out/
+frontend/next-env.d.ts
+frontend/tsconfig.json
+
+# production
+/build
+
+# misc
+.DS_Store
+*.pem
```

Comment:
There are some repeated values here

Reply:
I dont think so, already checked twice, there is some value that seems repeated, but not

```
/.next/
frontend/.next/
```